packages/synapse-react-client/src/components/FullWidthAlert/FullWidthAlert.tsx

```diff
@@ -145,14 +152,15 @@ function FullWidthAlert(props: FullWidthAlertProps) {
           tertiaryButtonConfig) && (
           <Stack
             spacing={{ xs: 1, lg: 2 }}
-            direction={{
-              xs: 'column-reverse',
-              sm: 'column',
-              lg: 'row',
-            }}
+            direction={{ xs: 'row', sm: 'column', lg: 'row' }}
             alignItems="center"
             display="flex"
             flexShrink={0}
+            sx={{
+              gap: { xs: '8px', sm: 'initial' },
+              flexWrap: { xs: 'wrap', sm: 'nowrap' },
+              width: { xs: '100%', sm: 'initial' },
```

Comment:
What do you think of adding this to make the buttons expand to full width at small screen sizes?
```suggestion
              width: { xs: '100%', sm: 'initial' },
              [theme.breakpoints.down('xs')]: {
                ["& > *"]: {
                  flexGrow: 1
                }
              }
```